src/styles/notification.css

```diff
@@ -95,10 +95,22 @@
   max-width: calc(100vw - 2em);
 }
 
+@media (min-width: 500px) {
```

Comment:
You use 500px here but in cy/tippy.js you use 650 to determine whether there is a small screen or not.  Should these two numbers be the same or not?